NEWS.md

```diff
@@ -1,5 +1,17 @@
 # fitzRoy development version
 
+# fitzRoy 1.4.1
```

Comment:
A small change. Could you merge in the latest version of NEWS.md (it won't be majorly different) and then just remove the "1.4.1" heading and put your updates in the "fitzRoy (development version)" section. That's where I keep anything that's in the Github version but not yet on a CRAN release